docs/src/index.md

```diff
@@ -0,0 +1,136 @@
+# ConservativeRegridding.jl
+
+ConservativeRegridding.jl provides functionality to regrid between two arbitrary grids.
+A grid is a tessellation of a space into polygons (or grid cells), each with an associated value.
+Data on a grid is referred to as a _field_, whereas the grid itself defines the tessellation.
+
+Regridding is performed **conservatively**, meaning the area-weighted mean is preserved.
+This is achieved by computing the intersection areas between all combinations of grid cells
+from the source and destination grids. These intersection areas provide the weights for
+averaging from neighboring cells.
+
+## Quick Start: SpeedyWeather Grid Transfer
+
+Here's an example of regridding between two different geodesic grids from [SpeedyWeather.jl](https://github.com/SpeedyWeather/SpeedyWeather.jl):
+
+```julia
+using SpeedyWeather
+using RingGrids
+using ConservativeRegridding
+
+# Helper function to get polygon faces for SpeedyWeather grids
+function get_faces(field)
+    grid = field.grid
+    Grid = typeof(grid)
+    nlat_half = grid.nlat_half
+    npoints = RingGrids.get_npoints2D(field)
+    E, S, W, N = RingGrids.get_vertices(Grid, nlat_half)
+    faces = Matrix{NTuple{2, Float64}}(undef, 5, npoints)
+    @inbounds for ij in 1:npoints
+        faces[1, ij] = (E[1, ij], E[2, ij])
+        faces[2, ij] = (S[1, ij], S[2, ij])
+        faces[3, ij] = (W[1, ij], W[2, ij])
+        faces[4, ij] = (N[1, ij], N[2, ij])
+        faces[5, ij] = (E[1, ij], E[2, ij])  # close the polygon
+    end
+    return faces
+end
+
+# Create random data on two different geodesic grid types
+field1 = rand(OctaHEALPixGrid, 24)
+field2 = rand(OctaminimalGaussianGrid, 24)
+
+# Get the polygon faces (vertices) for each grid
+faces1 = get_faces(field1)
+faces2 = get_faces(field2)
+
+# Build the regridder (internally converts vertices to polygons and fixes antimeridian crossings)
+R = ConservativeRegridding.Regridder(faces1, faces2)
+
+# Regrid from field2 to field1
+ConservativeRegridding.regrid!(field1, R, field2)
+
+# Regrid in the reverse direction using the transpose
+ConservativeRegridding.regrid!(field2, transpose(R), field1)
```

Comment:
@milankl is this example alright? I guess we would prefer `get_faces` to live in SpeedyWeather?